webview/src/stories/Setup.stories.tsx

```diff
@@ -98,17 +101,25 @@ CliFoundButNotCompatible.args = getUpdatedArgs({
 export const CannotInitializeGit = Template.bind({})
 CannotInitializeGit.args = getUpdatedArgs({
   canGitInitialize: false,
-  needsGitInitialized: true
+  needsGitInitialized: true,
+  projectInitialized: false
 })
 
 export const CanInitializeGit = Template.bind({})
 CanInitializeGit.args = getUpdatedArgs({
   canGitInitialize: true,
-  needsGitInitialized: true
+  needsGitInitialized: true,
+  projectInitialized: false
 })
 
 export const DvcUninitialized = Template.bind({})
 DvcUninitialized.args = getUpdatedArgs({
   canGitInitialize: undefined,
-  needsGitInitialized: undefined
+  needsGitInitialized: undefined,
+  projectInitialized: false
+})
+
+export const CliFoundManually = Template.bind({})
```

Comment:
Added a new story to show how things would look if python extension is not being used and replaced `NoCLIPythonExtensionNotUsed` and `NoCLIPythonExtensionUsed`with `NoCliPythonFound` since the two previous stories look the same now.